### ISSUE TITLE
Trends PNG export legend missing

### ISSUE THREAD

**Issue by pbstudent** [voyanttools/Voyant#12]
Exporting trends image produces a PNG without the legend that was displayed in Voyant Tools.
Workaround is to screencapture in Voyant Tools and cut/paste legend into exported image.

Wish list item: Trends export to SVG and CSV

**Comment by ajmacdonald** [comment]
Note to self: consider using https://github.com/niklasvh/html2canvas

**Comment by pbstudent** [comment]
Given Trends is a chart of numerical data, a PNG image seems less useful than SVG; whereas a spreadsheet web chart representation would be ideal.
However, Javascript screenshots could reduce the need for browser screenshot plugins or external software.

**Comment by ajmacdonald** [comment]
Hello pbstudent,
I'm not sure what you mean by "spreadsheet web chart".

I agree that an SVG export option would be useful. Note to self: https://docs.sencha.com/extjs/6.2.0/classic/Ext.chart.CartesianChart.html#cfg-engine

**Comment by pbstudent** [comment]
Online chart (e.g. Google Sheet or https://opensource.com/article/18/9/open-source-javascript-chart-libraries <https://opensource.com/article/18/9/open-source-javascript-chart-libraries>).

**Comment by pbstudent** [comment]
In addition,  https://analyticsindiamag.com/top-8-data-visualisation-libraries-in-javascript/ <https://analyticsindiamag.com/top-8-data-visualisation-libraries-in-javascript/>

**Comment by pbstudent** [comment]
Context is to get the Voyant Tools results exported to LibreOffice. The ideal situation is Trends and other results be able to have a direct ".ODS” file export format. Given Voyant Tools is open source, and the results are in plain text, ODS export could provide faster spreadsheet compilation and connectivity into an open source office suite.

**Comment by ajmacdonald** [comment]
Thank you for the context.
Unfortunately adding new export formats is not a big priority, however you can generally export in TSV format and then import that into LibreOffice.
Regarding Trends, it uses the same data as the "trends" column in the Terms panel, so you can export that data and then work with it elsewhere.

**Comment by pbstudent** [comment]
Yes I understand that you can re-do the chart, but given the software is designed to be as easy as possible for the user, a chart in Voyant Tools that could be exported to LO calc with corresponding data would expedite text analysis results to office workflow (i.e., WYSIWYG from web to LO calc).

However, such advanced data transfers across applications are not a deal breaker to using Voyant Tools. I could settle for SVG at this point in time.
